tests/e2e/local/minikube/README.md

```diff
@@ -16,19 +17,19 @@ You can run the following script to check/install of all pre-requisites, or use
 (This requires installation of [Homebrew](https://brew.sh) on macOS or debian based Linux distributions)
 
 ```bash
-sh install_prereqs_debian.sh
+. ./install_prereqs.sh
```

Comment:
does it require the sh files executable?

Reply:
Yes and I have made them executable.. we should do that for our vagrant scripts too I think..

Reply:
Please do =)